### PR TITLE
fix reference to $_KVM_RUNTIME_FOLDER_NAME

### PR DESCRIPTION
**Pull request by anurse**
This is what I get for making a "quick change to a variable name" AFTER running all the tests and then not running the tests again... I'll verify this shortly on a Unix machine, I'm on a Windows box at the moment.

### CODE REVIEW

**Comment by anurse** [comment]
Looks good now, with one minor caveat that I've followed up on via email.

File: src/kvm.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ _KVM_RUNTIME_SHORT_NAME="KRE"
 _KVM_RUNTIME_FOLDER_NAME=".k"
 _KVM_COMMAND_NAME="kvm"
 _KVM_VERSION_MANAGER_NAME="K Version Manager"
-_KVM_DEFAULT_FEED="https://www.myget.org/F/aspnetvnext/api/v2"
+_KVM_DEFAULT_FEED="https://www.myget.org/F/aspnetrelease/api/v2"
```
Comment (suhasj) [inline src/kvm.sh]: Need to change it at https://github.com/aspnet/dotnetsdk/blob/anurse-patch-1/src/kvm.sh#L32 as well

**Comment by suhasj** [comment]
looks fine to me :shipit: once the mentioned comment is fixed

**Comment by ChengTian** [comment]
Failed to build this branch on Mac. Some tests failed. Here is the values I used in tests:
```
[ -z "$KRE_FEED" ]          && export KRE_FEED="https://www.myget.org/F/aspnetrelease/api/v2" 
[ -z "$_TEST_VERSION" ]     && export _TEST_VERSION="1.0.0-beta3-11001"
[ -z "$_NUPKG_HASH" ]       && export _NUPKG_HASH="40aadb3c28e305df7f1cdfd4bcb5eb4c4b8032ce"
```

**Comment by suhasj** [comment]
is it failing to download latest package ? must be the issue of multiple packages on the feed

**Comment by ChengTian** [comment]
No, it can successfully download latest package. Looks like functional flaws. Here is a failed test case:
```
trace:       Setting alias 'test_alias_get' to 'kre-mono.1.0.0-beta3-11001'
error:     ✗ Aliases can be listed
error:       Setting alias 'test_alias_list_0' to 'kre-mono.1.0.0-beta3-11001'
error:       Setting alias 'test_alias_list_1' to 'kre-mono.1.0.0-beta3-11001'
error:       Setting alias 'test_alias_list_2' to 'kre-mono.1.0.0-beta3-11001'
error:       Setting alias 'test_alias_list_3' to 'kre-mono.1.0.0-beta3-11001'
error:       error: list did not include expected aliases
```

**Comment by anurse** [comment]
I'll take a look. Didn't have a good chance to verify on Mac yesterday.

**Pull request closed without merging**